edgexfoundry/developer-scripts
# cleanup: Remove edgex-volume and correct service dependencies

The purpose of this PR is to remove the ~100MB dependency of edgex-volume from the docker-compose files, and make modifications to the service dependencies to reflect the [service dependency graph](https://slack-files.com/TE66WRR3Q-F01148L6M7U-9a41e95836).  Several changes have been made to minimize spurious differences across mongo/redis, no-security/security, and amd64/arm64 versions of the compose files.  It is expected that these changes will also make it easier to remove the logging service in a future PR.

cc: @lenny-intel, @tingyuz, @andresrinivasan, @cloudxxx8 

### Testing instructions

Please ensure that you have the latest versions of all containers (`docker-compose -f whatever.yml pull`) and state (`docker volume prune`, `rm -fr /tmp/edgex/secrets`) before bringing up the stack.

Signed-off-by: Bryon Nevis <bryon.nevis@intel.com>
